@ndlib/gatsby-theme-marble/src/components/Layout/PageWrapper/NavigationHeader/LoginButton/index.js

```diff
@@ -4,13 +4,15 @@ import React, { useState } from 'react'
 import PropTypes from 'prop-types'
```

Comment:
This is a simple working example of how to load in the translation context. `useTranslation` is an implementation of React's `useContext` on the i18next context.